frontend/src/views/admin/assignments/offer-actions.js

```diff
@@ -114,34 +114,34 @@ function ConfirmWithDialogActionButtons(props) {
     return (
         <React.Fragment>
             <CreateOfferButtonWithDialog
-                actionButtonEnable={actionPermitted.canCreate}
+                actionButtonDisable={!actionPermitted.canCreate}
                 selectedAssignments={selectedAssignments}
-                actionCallback={createOffers}
+                callback={createOffers}
             />
             <WithdrawOfferButtonWithDialog
-                actionButtonEnable={actionPermitted.canWithdraw}
+                actionButtonDisable={!actionPermitted.canWithdraw}
```

Comment:
-> `disabled`. There is no point in having `actionButton` be part of this prop name.